lib/modules/platform/gerrit/client.ts

```diff
@@ -231,8 +231,15 @@ class GerritClient {
   ): string[] {
     const filterState = mapPrStateToGerritFilter(searchConfig.state);
     const filters = ['owner:self', 'project:' + repository, filterState];
-    if (searchConfig.branchName !== '') {
-      filters.push(`hashtag:sourceBranch-${searchConfig.branchName}`);
+    if (searchConfig.branchName) {
+      filters.push(
+        ...[
+          `footer:Renovate-Branch=${searchConfig.branchName}`,
+          // for backwards compatibility
+          'OR',
+          `hashtag:sourceBranch-${searchConfig.branchName}`,
```

Comment:
This will not work correctly without braces around the OR expression.
All filters after this OR will be optional......

Reply:
Nice catch!

Reply:
I tried reproducing it since I didn't notice anything wrong with it during my self-hosting, and indeed all filters after the OR ends up optional unless I add the braces.

Reply:
Thanks a lot, @NiasSt90!

Reply:
> I tried reproducing it since I didn't notice anything wrong with it during my self-hosting, and indeed all filters after the OR ends up optional unless I add the braces.

then you don't have autoapprove/automerge enabled... ;)